### PR TITLE
Release 1.198.1

### PR DESCRIPTION
**Pull request by meandavejustice**
## This pull request

- Release 1.198.0
- https://bugzilla.mozilla.org/show_bug.cgi?id=1686621

## Checklist

_Put an `x` in the boxes that apply_

- [x] My commit is GPG signed.
- [x] If applicable, I have modified or added tests which pass locally.
- [x] I have added necessary documentation (if appropriate).
- [x] I have verified that my changes render correctly in RTL (if appropriate).

### CODE REVIEW

**Review by chenba — APPROVED** [review]

**Comment by dannycoates** [comment]
Don't merge this for a min, until after 197.3 thx

**Comment by dannycoates** [comment]
I'll fix these conflicts

**Comment by vasilicatamas** [comment]
Finalized testing the End to End suite for 1.198.1 Train and sent a Green Signoff via email.

**Pull request merged**
